src/tests/Operate/operate.test.js

```diff
@@ -0,0 +1,38 @@
+import Big from 'big.js';
+import operate from '../../logic/operate';
+
+describe('operate', () => {
+  it('adds two numbers correctly', () => {
+    const result = operate('2', '3', '+');
+    expect(result).toEqual(Big('5').toString());
+  });
+
+  it('subtracts two numbers correctly', () => {
```

Comment:
- [ ] Great job by adding the snapshot but kindly follow the project requirement also according to your previouse reviewer All the functionalities & scenarios of the operate.js and calculate.js modules should be covered using Jest.
There should be the usage of Jest snapshots for testing all the components.kindly add the snapshot test to the remain component whch operate.js 
There should be the usage of the React Testing Library to simulate user interaction.